core/src/main/scala/org/apache/spark/MapOutputTracker.scala

```diff
@@ -335,28 +335,12 @@ private[spark] abstract class MapOutputTracker(conf: SparkConf) extends Logging
    *         tuples describing the shuffle blocks that are stored at that block manager.
    */
   def getMapSizesByExecutorId(
```

Comment:
can we keep it as overload as well?

Reply:
It would be only used by 3 tests yet. Do you think it's necessary?